include/envoy/config/grpc_mux.h

```diff
@@ -125,6 +125,16 @@ class GrpcMux {
                                   const std::set<std::string>& resources,
                                   SubscriptionCallbacks& callbacks,
                                   std::chrono::milliseconds init_fetch_timeout) PURE;
+
+  /**
+   * The only difference between addToWatch() and addOrUpdateWatch() is that the 'resources' here
```

Comment:
I think I would phrase this comment a bit differently, something like: 'addToWatch adds additional resources to the watch. Unlike addOrUpdateWatch, it never removes resources'

Reply:
> I think I would phrase this comment a bit differently, something like: 'addToWatch adds additional resources to the watch. Unlike addOrUpdateWatch, it never removes resources'

OK, I'll fix it.